main.go

```diff
@@ -21,11 +22,11 @@ var localCert []byte
 var localKey []byte
 
 func main() {
-	message.SetLogLevel(message.DebugLevel)
+	slog.SetLogLoggerLevel(slog.LevelDebug)
 
 	r, inCluster, err := api.Setup(&assets)
 	if err != nil {
-		message.WarnErr(err, "failed to start the API server")
+		slog.Warn(fmt.Sprintf("failed to start the API server: %s", err))
```

Comment:
`slog` enables us to have structured logs, so we should try to use that here. 
```suggestion
		slog.Warn("failed to start the API server", "error", err)
```

Playground example of the differences:

- https://go.dev/play/p/PPRFLglkj1h

`log/slog` docs: 
- https://pkg.go.dev/log/slog#pkg-overview